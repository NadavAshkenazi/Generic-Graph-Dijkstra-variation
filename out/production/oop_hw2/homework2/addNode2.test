CreateGraph X
CreateNode b 3
CreateNode c 2
CreateNode a 4
AddNode X c
AddNode X a
AddNode X b
ListNodes X
