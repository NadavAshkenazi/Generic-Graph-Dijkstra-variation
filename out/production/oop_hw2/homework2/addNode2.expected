created graph X
created node b with cost 3
created node c with cost 2
created node a with cost 4
added node c to X
added node a to X
added node b to X
X contains: a b c
